command-line/dot-profile.md

```diff
@@ -142,10 +135,6 @@ Nano is a Linux command line text editor. It is relatively easy to learn and use
 - Any command prefixed with the caret symbol(^) means to use the "ctrl" key. For example, **^G** means to press **ctrl+G** keys at he same time.
 - Any command prefixed with letter M means to press the Alt key. For example, **M-R** means to press **Alt+r** keys together at the same time.
 
-For more detailed info on how to use the nano tool, visit these sites below:
```

Comment:
I moved them to supplemental resources as description to use this material is not provided.
Is it ok or shall I place them here again.